package.json

```diff
@@ -14,9 +14,10 @@
     "express-skeleton",
     "express-js"
   ],
-  "author": "John Doe",
+  "author": "Riccardo Maldera",
```

Comment:
solo quando sei certo di aver fatto meglio del proprietario del repo puoi far richiesta di aggiungerti come autore, il cambio autore in un progetto non è mai ben visto.

Reply:
okay, grazie per il consiglio